### PR TITLE
[CLOUD-390] Update ubuntu and centos image OS versions

### PR DESCRIPTION
**Pull request by gibchikafa**

### CODE REVIEW

**Review by o-alex — COMMENTED** [review]
We should probably add a new folder and not change the current ubuntu. Does this still work for ubuntu 18? Otherwise this might fail current master vms?

File: docker/ubuntu/Dockerfile [hunk 1]
```diff
@@ -19,6 +19,10 @@ update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 60 --slave /usr/bi
 update-alternatives --config gcc; \
 apt-get install -y wget openssl libssl-dev git make
 
+# will put repo.hops.works
+RUN wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+RUN -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+
```
Comment (maismail) [inline docker/ubuntu/Dockerfile:25]: this won't work and i don't think it would be needed for building ePipe

File: docker/ubuntu/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ARG ubuntu_image=bionic
+ARG ubuntu_image=22.04
```
Comment (maismail) [inline docker/ubuntu/Dockerfile:1]: same here, I expect that multiple steps of the building process would needs to be adapted to 22.04. The best way is to try it on a 22.04 ubuntu vm then update the docker image. I can give it a try

**Review by maismail — CHANGES_REQUESTED** [review]

File: docker/ubuntu/Dockerfile (hunk 2 above)
Comment (gibchikafa) [reply docker/ubuntu/Dockerfile:1]: I tried it was able to build but you can try to build.

File: docker/ubuntu/Dockerfile (hunk 1 above)
Comment (gibchikafa) [reply docker/ubuntu/Dockerfile:25]: Don't you run epipe in a docker container sometimes?

File: docker/ubuntu/Dockerfile (hunk 2 above)
Comment (maismail) [reply docker/ubuntu/Dockerfile:1]: did you try by running `./docker-build.sh`? you need to change the image name also in that file as well 
https://github.com/logicalclocks/ePipe/blob/master/docker-build.sh#L20

Comment (gibchikafa) [reply docker/ubuntu/Dockerfile:1]: yeah I build it using `docker-build.sh `script

Comment (gibchikafa) [reply docker/ubuntu/Dockerfile:1]: I see the `ubuntu_image` argument in the script.  But it appeared to be building from ubuntu22. I will try again.

Comment (gibchikafa) [reply docker/ubuntu/Dockerfile:1]: why do we build from `focal ` on arm?

Comment (maismail) [reply docker/ubuntu/Dockerfile:1]: I was only using it for testing on mac m1

**Comment by maismail** [comment]
Jenkins test this please

**Comment by maismail** [comment]
Jenkins test this please

**Comment by maismail** [comment]
Jenkins test this please

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**